lib/truffler.js

```diff
@@ -141,16 +141,21 @@ Truffler.prototype._run = function(url, options, done) {
 		// Open the URL in PhantomJS
 		openUrl: guardAgainstTimeout(function(next) {
 			options.log.debug('Opening "' + url + '" in PhantomJS');
+
+			state.page.onResourceError = function(resourceError) {
+				this.reason = resourceError.errorString;
```

Comment:
I worked with @moddular on that bit of code, we tried using a var in the beginning and it wouldn't work so ended up with the rather odd bind and this.